mocha.js

```diff
@@ -776,7 +776,7 @@ module.exports = function(suite) {
   });
```

Comment:
@mochajs/core I’m not sure what to do about these changes here. Is it all right to include them in the commit? 😳